src/partisan_static_peer_service_manager.erl

```diff
@@ -164,21 +161,14 @@ handle_call({reserve, _Tag}, _From, State) ->
 handle_call({leave, _Node}, _From, State) ->
     {reply, error, State};
 
-handle_call({join, {Name, _, _}=Node},
+handle_call({join, {_, _, _}=Node},
             _From,
-            #state{pending=Pending0, connections=Connections0}=State) ->
-    %% Attempt to join via disterl for control messages during testing.
-    _ = net_kernel:connect(Name),
-
-    %% Add to list of pending connections.
-    Pending = [Node|Pending0],
```

Comment:
What's the motivation for removing the pending connections?  This is so things get eventually retried.